test/Microsoft.ML.Functional.Tests/Common.cs

```diff
@@ -267,5 +267,20 @@ public static void AssertMetricsStatistics(RegressionMetricsStatistics metrics)
             AssertMetricStatistics(metrics.RSquared);
             AssertMetricStatistics(metrics.LossFn);
         }
+
+        /// <summary>
+        /// Verify that a numerical array has no NaNs or infinities.
+        /// </summary>
+        /// <param name="array">An array of doubles.</param>
+        public static void AssertFiniteNumbers(double[] array, int ignoreElementAt = -1)
```

Comment:
>AssertFiniteNumbers [](start = 27, length = 19)

Where is this function being used?

Reply:
It's used here IntrospectGamShapeFunctions

---
In reply to: [262695483](https://github.com/dotnet/machinelearning/pull/2859#discussion_r262695483) [](ancestors = 262695483)

Reply:
That's right. I put it in Common because I imagine that I'll use it again. Although `ignoreElementAt` is definitely a binning-only kind of thing.

---
In reply to: [262705673](https://github.com/dotnet/machinelearning/pull/2859#discussion_r262705673) [](ancestors = 262705673,262695483)